addons/website_sale_comparison/views/snippets.xml

```diff
@@ -14,6 +14,12 @@
                    data-no-preview="true"
                    data-reload="/"/>
     </xpath>
+    <xpath expr="//we-button[hasclass('o_we_buy_now_btn')]" position="after">
+        <we-button title="Compare" class="fa fa-fw fa-exchange"
```

Comment:
Not 100% sure, but shouldn't you reexport the pot to make sure the `Compare` title is translated ? :thinking: